communication/nats/connection_wrap.go

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
```

Comment:
There is no need to update old files headers.

Reply:
As I understand date is when file was created. For example https://github.com/golang/go/blob/master/src/runtime/alg.go#L1 alg.go created on 2014 and modified 29 days ago.